javascript/frameworks/cap/test/queries/cqlinjection/srv/service1.js

```diff
@@ -0,0 +1,786 @@
+      await INSERT.into(Service1Entity).entries("ID =" + id);
+    });
+
+    this.on("send00222", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await INSERT.into(Service1Entity).entries(`ID =` + id);
+    });
+
+    this.on("send00223", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await INSERT.into(Service1Entity).entries(`ID = ${id}`);
+    });
+
+    this.on("send00224", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await INSERT.into(Service1Entity).entries`ID = ${id}`; // FP
+    });
+
+    this.on("send00231", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPDATE.entity(Service1Entity).set("col1 = col1 + " + id).where("ID =" + id);
+    });
+
+    this.on("send00232", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPDATE.entity(Service1Entity).set("col1 = col1 + " + id).where(`ID =` + id);
+    });
+
+    this.on("send00233", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPDATE.entity(Service1Entity).set("col1 = col1 + " + id).where(`ID = ${id}`);
+    });
+
+    this.on("send00234", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPDATE.entity(Service1Entity).set("col1 = col1 + " + id).where`ID = ${id}`; // FP
+    });
+
+    this.on("send00241", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPSERT.into(Service1Entity).entries({ id: "" + id });
+    });
+
+    this.on("send00242", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPSERT.into(Service1Entity).entries({ id: `` + id });
+    });
+
+    this.on("send00243", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await UPSERT.into(Service1Entity).entries({ id: `${id}` });
+    });
+
+    this.on("send00251", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await DELETE.from(Service1Entity).where("ID =" + id);
+    });
+
+    this.on("send00252", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await DELETE.from(Service1Entity).where(`ID =` + id);
+    });
+
+    this.on("send00253", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await DELETE.from(Service1Entity).where(`ID = ${id}`);
+    });
+    
+    this.on("send00254", async (req) => {
+      const { id } = req.data;
+      const { Service1Entity } = this.entities;
+      await DELETE.from(Service1Entity).where`ID = ${id}`; // FP
```

Comment:
No longer an FP?

Reply:
Same strategy as [this comment](https://github.com/advanced-security/codeql-sap-js/pull/200#discussion_r2178535082) will be used and will be addressed together.